lib/classes/PluginManager.js

```diff
@@ -352,6 +353,13 @@ class PluginManager {
     });
   }
 
+  assignDefaultOptions(command) {
+    _.forEach(command.options, (value, key) => {
+      if (value.default && (!this.cliOptions[key] || this.cliOptions[key] === true)) {
```

Comment:
😂 thanks for the response @dougmoscrop.

So I would say that we can get rid of the double check since they really both check the same.